### PR TITLE
Update QSPI for new SPI Flash

### PR DESCRIPTION
**Pull request by ladyada**
this allows us to have dynamic SD/Flash selection. amazing!

### CODE REVIEW

**Comment by Tijgerd** [comment]
@ladyada I think this pullrequest for SdFat will fix the builds https://github.com/adafruit/SdFat/pull/3

File: Adafruit_Arcada_Filesystem.cpp [hunk 1]
```diff
@@ -1,62 +1,78 @@
...
 
+Adafruit_FlashTransport_QSPI flashTransport(PIN_QSPI_SCK, PIN_QSPI_CS, PIN_QSPI_IO0, PIN_QSPI_IO1, PIN_QSPI_IO2, PIN_QSPI_IO3);
+Adafruit_SPIFlash Arcada_QSPI_Flash(&flashTransport);
+FatFileSystem Arcada_QSPI_FileSys;
+
 /**************************************************************************/
 /*!
     @brief  Initialize the filesystem, either SD or QSPI
-    @return True if a filesystem exists and was found
+    @params desiredFilesys The filesystem we'd prefer to use, can be  ARCADA_FILESYS_SD, ARCADA_FILESYS_QSPI, or ARCADA_FILESYS_SD_AND_QSPI
```
Comment (Tijgerd) [inline Adafruit_Arcada_Filesystem.cpp]: should be @param

File: Adafruit_Arcada_USBMSD.cpp [hunk 2]
```diff
@@ -6,82 +6,85 @@ static uint32_t last_access_ms;
...
-  int32_t msc_write_cb (uint32_t lba, uint8_t* buffer, uint32_t bufsize);
-  int32_t msc_read_cb (uint32_t lba, void* buffer, uint32_t bufsize);
-  void msc_flush_cb (void);
-#endif // SD or QSPI
-#endif // TinyUSB
 
 /**************************************************************************/
 /*!
     @brief  Make the raw filesystem of the Arcada board available over USB
     @return True on success, false on failure
```
Comment (Tijgerd) [inline Adafruit_Arcada_USBMSD.cpp:33]: add:
    @param desiredFilesys The filesystem we'd prefer to use, can be  ARCADA_FILESYS_SD, ARCADA_FILESYS_QSPI, or ARCADA_FILESYS_SD_AND_QSPI

**Review by Tijgerd — CHANGES_REQUESTED** [review]
Just a single typo-fix and added comment and it should be good to go :)

File: Adafruit_Arcada_USBMSD.cpp (hunk 2 above)
Comment (ladyada) [reply Adafruit_Arcada_USBMSD.cpp:33]: yep was asleep :)

**Pull request merged**